src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -109,16 +111,38 @@ public static Email parseEmail(String email) throws ParseException {
     public static Star parseStar(String star) throws ParseException {
         requireNonNull(star);
         String trimmedStar = star.trim(); // trim the string to only get the number
-        if (!StringUtil.isUnsignedInteger(trimmedStar)) {
+        if (!StringUtil.isNonZeroUnsignedInteger(trimmedStar))  { // star command only allows non-zero
```

Comment:
can i check why this if statement is necessary when line 118 is a more restrictive version of this?